### ISSUE TITLE
Removing CSS stylesheet

### ISSUE THREAD

**Issue by chvillanuevap** [bjorsq/rich-text-excerpts#3]
Hi there!

I would like to dequeue the CSS stylesheet this plugin uses. Right now, it is very difficult to do so as the stylesheet get hooked on `admin_print_styles`, which is turn gets hooked on `admin_menu`. 

So calling `wp_dequeue_style` on `admin_print_styles` won't do the trick. You have to wait for `admin_menu` to get called, and then hook it there after `plugin_admin_styles` gets hooked on `admin_print_styles`. 

Or you could add an option to dequeue scripts and styles.

Do you accept pull requests?

**Comment by bjorsq** [comment]
Hi Hernán,

I've been sitting on some updates for this plugin for a while now which is why it hadn't been updated for so long (I'd even forgotten how to use subversion!). I decided I should update it when I got your message - the plugin was originally written some years ago when the official advice was to enqueue scripts and styles like this - I realise this isn't the case any more.

I've released an update today which enqueues scripts and styles using the admin_enqueue_scripts hook:

1. RichTextExcerptsAdminScript - should be included on the plugin options page (shows/hides options and errors)
2. RichTextExcerptsEditorScript - should be included on all editor pages for supported post types (handles dragging editor metabox)
3. RichTextExcerptsAdminCSS - should be included on all editor pages for supported post types (some style changes for editor metabox)

You should be able to hook into admin_enqueue_scripts later (with a priority > 10) and remove them. I think of these three files, the admin CSS is the only one which isn't strictly neccessary - most of the changes here were suggested by another user.

Yes, I do accept pull requests - if you have a better way of handling the styling in the editor metabox I'd be interested in your input.

Best wishes,

peter (bjorsq)

**Comment by chvillanuevap** [comment]
Thank you for the update, Peter! I had dequeued the stylesheet, but first I had to hook a function to `admin_menu`, and then another one to `admin_print_styles`. Your changes will make dequeuing the stylesheet so much easier!

It's funny, I was looking around my own theme and noticed that I was also using `admin_print_styles` to enqueue styles instead of `admin_enqueue_scripts`. You are right -- I think they changed their official advice from a few years ago.

Anyway, I'll update my function to match your changes. Thanks a lot!
